frontend/src/styles.css

```
@@ -257,13 +259,14 @@ h3 {
   filter: brightness(1.16) saturate(1.08);
 }
 
-.hero-title {
+.hero-title,
+.upload-home-brand__title {
   margin-top: 0;
   font-family: "Sora", ui-sans-serif, system-ui, sans-serif;
```

Stylelint (17.6.0)

[error] Unexpected quotes around "Sora" (font-family-name-quotes)

(font-family-name-quotes)